src/server/pkg/storage/fileset/pg_store_test.go

```diff
@@ -0,0 +1,14 @@
+package fileset
+
+import (
+	"testing"
+
+	"github.com/pachyderm/pachyderm/src/server/pkg/dbutil"
+)
+
+func TestPGStore(t *testing.T) {
```

Comment:
`PG` -> `Postgres`. This change should be applied across all names (including files).